### PR TITLE
Update configure-server-endpoints.md

### PR DESCRIPTION
**Pull request by msbemba**
Updated the document to remove - `OptionalParamsPolicy` - as it is not available  in the onboarding package for windows Server 2012 and 2016,

Fixes https://github.com/MicrosoftDocs/microsoft-365-docs-pr/issues/23604

### CODE REVIEW

**Comment by msbemba** [comment]
@JohanFreelancer9 Please copy review

File: microsoft-365/security/defender-endpoint/configure-server-endpoints.md [hunk 1]
```diff
@@ -157,7 +157,6 @@ The **installation package** contains an MSI file that installs the Microsoft De
 
 The **onboarding package** contains the following files:
```
Comment (JohanFreelancer9) [inline microsoft-365/security/defender-endpoint/configure-server-endpoints.md]:
```suggestion
The **onboarding package** contains the following file:
```

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review.

**Comment by yogkumgit** [comment]
@JohanFreelancer9

**Comment by yogkumgit** [comment]
@mjcaparas Please merge this PR. Thanks

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by yogkumgit** [comment]
@mjcaparas Gentle reminder.

**Pull request merged**